opentracing-api/src/main/java/io/opentracing/propagation/Format.java

```diff
@@ -71,5 +71,16 @@
          * @see Format
          */
         public final static Format<ByteBuffer> BINARY = new Builtin<ByteBuffer>();
+
+        /**
+         * @return Short name for built-in formats as they tend to show up in exception messages.
+         */
+        @Override
+        public String toString() {
```

Comment:
I think it would be cleaner to define a private constructor that accepts a `String name` parameter and toString simply returns the `name`. It would also fix the leak in this class - its called "Buildin", but people can create more instances of it, which wasn't the intention.